src/Classes/ServiceAPI/MyRadio_Webcam.php

```diff
@@ -69,32 +69,39 @@ public static function getArchiveTimeRange()
      */
     public static function getCurrentWebcam()
     {
-        $current = file_get_contents(Config::$webcam_current_url);
+        if (Config::$webcam_current_url) {
+            $current = (int)file_get_contents(Config::$webcam_current_url);
```

Comment:
It won't throw an exception - in this case the value will be 0 (see http://php.net/manual/en/language.types.string.php#language.types.string.conversion)